.gitignore

```diff
@@ -2,4 +2,5 @@ elm-stuff/
 repl-temp-*
 npm-debug.log
 node_modules/
-dist/
\ No newline at end of file
+dist/
+demo/index.html
```

Comment:
Didn't we just delete that one? 🤔

Reply:
I accidentally committed it to begin with. Just making sure I don't do that again.